stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/actions/data/sql/actions/SQLDatasourceUtils.java

```diff
@@ -607,6 +610,26 @@ public static void setNClobValue(PreparedStatement stmt, BValue value, int index
         }
     }
 
+    public static void setRefCursorValue(Connection connection, PreparedStatement stmt, int index, int direction) {
+        try {
+            if (Constants.QueryParamDirection.OUT == direction) {
+                if (ORACLE_DATABASE_NAME.equals(connection.getMetaData().getDatabaseProductName())) {
```

Comment:
Databasae name is already there in SQLDatasource.databaseName. Shall we use it instead of getting meta data again?

Reply:
+1. Will fix this